bff/src/Bff.Blazor/BffBuilderExtensions.cs

```diff
@@ -14,7 +14,7 @@ public static BffBuilder AddBlazorServer(this BffBuilder builder)
         builder.Services
             .AddOpenIdConnectAccessTokenManagement()
             .AddBlazorServerAccessTokenManagement<ServerSideTokenStore>()
-            .AddScoped<AuthenticationStateProvider, BffServerAuthenticationStateProvider>();
+            .AddTransient<IClaimsTransformation, AddServerManagementClaimsTransform>();
```

Comment:
Does the transformation need to be created on demanding this, or could it be singleton? I guess this way you get an up to date options instance in the transformation, but maybe that could use an options monitor to get the same behavior.

Reply:
Also works.